Displaying an extra notice when inotify runtime support is enabled.
=====================================================================
--- ./configure.ac.bak	2008-03-02 14:32:08.000000000 -0500
+++ ./configure.ac	2008-03-02 14:33:04.000000000 -0500
@@ -2933,6 +2933,7 @@
         # if system inotify header exists, go with it, otherwise take our own
         if test "x$ac_cv_header_sys_inotify_h" = xyes; then
              AC_DEFINE([SYS_INOTIFY_H_OK],[1],[sys/inotify.h exists])
+             AC_MSG_NOTICE([using system inotify headers])
         fi
         AC_DEFINE([HAVE_INOTIFY], [1], [inotify presence])
         AC_SUBST(INOTIFY_CXXFLAGS)
